### PR TITLE
Update usage-history.md

### PR DESCRIPTION
**Pull request by jprcoliveira**
## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [x] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/console/manage-organization/usage-history.md [hunk 1]
```diff
@@ -5,11 +5,17 @@ description: "Let's observe our workflow usage."
 ---
 
 :::note
-This setting is only visible during the **Trial Phase** for owners and admins.
+The usage history is visible only to owners and admins.
```
Comment (ultraschuppi) [inline docs/components/console/manage-organization/usage-history.md:8]: well, the `Usage` tab is only available in non-paying salesplans - so this is is not too cowrecked i think?
for paying orgs it is in `Billing`

File: docs/components/console/manage-organization/usage-history.md [hunk 2]
```diff
@@ -5,11 +5,17 @@ description: "Let's observe our workflow usage."
...
 :::
 
-Two key metrics play a role in paid plans: the number of started process instances, and the number of claimed user tasks. The **Organization Management** for trial plans provides a usage view for these metrics across the organization.
+Three key metrics play a role in paid plans: the number of started process instances, decision instances, and the number of claimed tasks users. The **Organization Management** provides a usage view for these metrics across the organization.
```
Comment (christinaausley) [inline docs/components/console/manage-organization/usage-history.md:11]: Do do mean "number of claimed user tasks" here?

**Review by christinaausley — DISMISSED** [review]
One comment for clarification -- thank you for this!

**Comment by akeller** [comment]
What's the status of this PR? It looks like there is still some feedback that hasn't been incorporated.

**Comment by jprcoliveira** [comment]
@akeller and @christinaausley updated the wording. Tests running. Any released planned for today where we can include this?

**Review by akeller — APPROVED** [review]

**Pull request merged**